server/controllers/AnswerController.js

```diff
@@ -12,51 +12,51 @@ export default class AnswerController {
    * @description This method handles post answer
    * @param {object} request request object
    * @param {object} response response object
-   *
    * @returns {Object} Object
   */
   static async postAnswer(request, response) {
     const userId = request.userId.id;
     const { questionId } = request.params;
     const { answerBody } = request.body;
     const checkQuestion = await db.query(checkQuestionId(questionId));
-    if (checkQuestion.rowCount > 0) {
-      console.log(postAnswer(answerBody, userId, questionId));
-      const result = await db.query(postAnswer(answerBody, userId, questionId)).catch(error => error.message);
-      console.log(result);
-      if (result.rowCount > 0) {
-        return response.status(201).json({
-          status: 'success',
-          message: 'Answer has been posted successfully',
-          answer: result.rows,
+    try {
+      if (checkQuestion.rowCount > 0) {
+        const result = await db.query(postAnswer(answerBody, userId, questionId));
+        if (result.rowCount > 0) {
+          return response.status(201).json({
+            status: 'success',
+            message: 'Answer has been posted successfully',
+            answer: result.rows,
+          });
+        }
+        return response.status(500).json({
```

Comment:
You need one `500` error, and the one in the catch block would suffice. This block of code doesn't need to be here